src/sdk/bucketspace_fs.js

```diff
@@ -85,6 +85,14 @@ class BucketSpaceFS extends BucketSpaceSimpleFS {
             if (account.nsfs_account_config.distinguished_name) {
                 account.nsfs_account_config.distinguished_name = new SensitiveString(account.nsfs_account_config.distinguished_name);
             }
+            try {
+                const path_for_account_or_user_config_file = access_key === anonymous_access_key ?
+                this.config_fs.get_account_path_by_name(config.ANONYMOUS_ACCOUNT_NAME) :
```

Comment:
this logic should be inside stat_account_config_file()

Reply:
I tried to avoid passing the Symbol type that might be in access_key.
Anyway, now I changed it according to this comment, thanks.